packages/wallet-client/src/interface/from-wallet.ts

```diff
@@ -11,10 +11,11 @@ export const fundingSuccess = (channelId, position: string) => ({
   channelId,
   position,
 });
-export const fundingFailure = (channelId, reason) => ({
+export const fundingFailure = (channelId: any, reason: 'FundingDeclined' | 'Other', error?: string) => ({
```

Comment:
Does the `| 'Other'` here have any effect?

Reply:
It should limit `reason` to be either the string `Other` or `FundingDeclined`. It makes it easier to check for specific types of failures instead of trying to parse  the error message.